Description: Create a Media description section in manpages
 The manpages setfdprm(1) and superformat(1) make a reference to an
 inexistant "Media description" section. That is because there is no
 generated manpage from mediaprm.texi.
 This patch adds the missing "Media description" sections, that cross-
 reference the Texinfo (or HTML, or DVI) documentation.
Author: Matteo Cypriani <mcy@lm7.fr>
Bug-Debian: http://bugs.debian.org/282014
--- a/doc/setfdprm.1
+++ b/doc/setfdprm.1
@@ -118,6 +118,16 @@
 Disables format detection messages for the specified auto-configuring
 floppy device.
 .PP
+.SH Media description
+Please see the Media description section in the full fdutils
+documentation:
+.TQ
+- Texinfo documentation (info fdutils)
+.TQ
+- HTML documentation in /usr/share/doc/fdutils/Fdutils.html
+.TQ
+- or DVI documentation in /usr/share/doc/fdutils/Fdutils.dvi.gz
+.PP
 .SH Bugs
 This documentation is grossly incomplete.
 .SH See Also
--- a/doc/superformat.1
+++ b/doc/superformat.1
@@ -293,6 +293,16 @@
 sectors. In reality, superformat expects the skews to be expressed in
 raw bytes.
 .PP
+.SH Media description
+Please see the Media description section in the full fdutils
+documentation:
+.TQ
+- Texinfo documentation (info fdutils)
+.TQ
+- HTML documentation in /usr/share/doc/fdutils/Fdutils.html
+.TQ
+- or DVI documentation in /usr/share/doc/fdutils/Fdutils.dvi.gz
+.PP
 .SH Examples
 In all the examples of this section, we assume that drive 0 is a 3 1/2
 and drive 1 a 5 1/4.
